cmd/ndm_daemonset/filter/osdiskexcludefilter.go

```diff
@@ -109,7 +113,18 @@ func (odf *oSDiskExcludeFilter) Include(d *controller.DiskInfo) bool {
 	return true
 }
 
-// Exclude returns true if disk devpath matches with excludeDevPath
+// Exclude returns true if disk devpath does not match with excludeDevPath
```

Comment:
This comment looks confusing to me. Earlier comment seemed relevant with method signature.

Reply:
@AmitKumarDas  The earlier comment was wrong. `Include` returns true if path matches, `Exclude` returns true if path does not match. That is how it works.